src/frontend/src/components/MyFeedsPage/ExistingFeedList.jsx

```diff
@@ -1,11 +1,39 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { TextValidator } from 'react-material-ui-form-validator';
-import { Grid, Typography } from '@material-ui/core';
+import { Grid, TextField, Typography } from '@material-ui/core';
 import { AccountCircle, RssFeed } from '@material-ui/icons';
+import useSiteMetadata from '../../hooks/use-site-metadata';
 import DeleteFeedDialogButton from './DeleteFeedDialogButton.jsx';
 
-function ExistingFeedList({ feedHash }) {
+function ExistingFeedList({ userInfo }) {
+  const [feedHash, updateFeedHash] = useState({}); // { id1: {author: '...', url: '...'}, id2: {...}, ... }
+  const { telescopeUrl } = useSiteMetadata();
+
+  useEffect(() => {
+    if (userInfo.id) {
+      (async function hashUserFeeds() {
+        try {
+          const response = await fetch(`${telescopeUrl}/user/feeds`);
+
+          if (!response.ok) {
+            throw new Error(response.statusText);
+          }
+
+          const userFeeds = await response.json();
+          const userFeedHash = userFeeds.reduce((hash, feed) => {
+            hash[feed.id] = { author: feed.author, url: feed.url };
+            return hash;
+          }, {});
+
+          updateFeedHash(userFeedHash);
+        } catch (error) {
+          console.log('Error hashing user feeds', error);
+          throw error;
```

Comment:
No point throwing this, as there's nothing to catch it.

Reply:
Removed `throw` call in https://github.com/Seneca-CDOT/telescope/pull/994/commits/8a099c8ad1df285770020ddd3860102cd9ecc057